From 70d5fe78b4b501c4f23ee88776c723d5c2d9a4a4 Mon Sep 17 00:00:00 2001
From: Aichun Shi <aichun.shi@intel.com>
Date: Thu, 8 Dec 2022 15:02:47 +0800
Subject: [PATCH 0876/1646] [Backport] Enable Intel In Field Scan(IFS) as
 kernel module

Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I651S7

Reference: https://gitee.com/openeuler/kernel/commit/53c13478c34c44c99e35129156659edf16af4132

--------------------------------------------

Enable Intel In Field Scan(IFS) as kernel module in openeuler_defconfig.

Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index a2f1c15d746e..fa258d271c27 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -6492,6 +6492,7 @@ CONFIG_XEN_FRONT_PGDIR_SHBUF=m
 # CONFIG_GREYBUS is not set
 # CONFIG_STAGING is not set
 CONFIG_X86_PLATFORM_DEVICES=y
+CONFIG_INTEL_IFS=m
 CONFIG_ACPI_WMI=m
 CONFIG_WMI_BMOF=m
 # CONFIG_ALIENWARE_WMI is not set
-- 
2.17.1

